workloads/aws-oidc/aws-oidc-analysis/deploy.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: v1
```

Comment:
There are various changes in the AWS multi-mesh demo PR to bring in, but that can be a follow up.